.dev/sass/compat/woocommerce.scss

```diff
@@ -37,3 +37,157 @@ body.single-product {
 	}
 
 }
+
+#woocommerce-cart-menu-item {
```

Comment:
@EvanHerman I'm not sure about venturing into the `woocommerce` namespace. What do you think?